database/CoreDatabase.java

```diff
@@ -143,7 +143,7 @@ protected CoreDatabase(CoreDatabaseManager databaseMgr, String name, Factory.Ses
         statisticsCorrector = createStatisticsCorrector();
         sessions = new ConcurrentHashMap<>();
         rocksConfiguration = new RocksConfiguration(options().storageDataCacheSize(),
-                options().storageIndexCacheSize(), LOG.isDebugEnabled(), ROCKS_LOG_PERIOD);
+                options().storageIndexCacheSize(), LOG.isDebugEnabled() || LOG.isTraceEnabled(), ROCKS_LOG_PERIOD);
```

Comment:
periodic logging of the rocks properties should be done on trace or debug - we had assumed `isDebugEnabled()` included trace transitively